scoreboard/models.py

```diff
@@ -507,11 +505,12 @@ def set_attachments(self, attachments):
             aid_set.add(a['aid'])
             attachment = Attachment.query.get(a['aid'])
             if not attachment:
-                Attachment.create(a['aid'], a['filename'], a['content_type'], self)
+                logging.warning('Trying to add attachment that does not exist')
```

Comment:
Let's log the attachment id as well, as that will help debugging issues that may occur.